internal/agent/hooks/kcrypt_uki.go

```diff
@@ -163,33 +161,16 @@ func (k KcryptUKI) Run(c config.Config, _ v1.Spec) error {
 		return err
 	}
 
-	// Copy logs to persistent partition
-	c.Logger.Debug("Copying logs to persistent partition")
-	err = machine.Mount(constants.PersistentLabel, constants.PersistentDir)
-	if err != nil {
-		c.Logger.Errorf("could not mount persistent partition: %s", err)
-		return nil
-	}
-	varLog := filepath.Join(constants.PersistentDir, ".state", "var-log.bind")
-	// Create the directory on persistent
-	err = fsutils.MkdirAll(c.Fs, varLog, 0755)
-	if err != nil {
-		c.Logger.Errorf("could not create directory on persistent partition: %s", err)
-		return nil
-	}
-	// Copy all current logs to the persistent partition
-	err = internalutils.SyncData(c.Logger, c.Runner, c.Fs, "/var/log/", varLog, []string{}...)
+	c.Logger.Logger.Debug().Msg("Finish KcryptUKI hook")
+	// We now have the partitions unlocked and ready, lets call the other hooks here instead of closing and reopening them each time
+	err = BundlePostInstall{}.Run(c, spec)
 	if err != nil {
-		c.Logger.Errorf("could not copy logs to persistent partition: %s", err)
-		return nil
+		return err
 	}
-	err = machine.Umount(constants.PersistentDir)
+	_ = CustomMounts{}.Run(c, spec)
```

Comment:
this is the same than the mount/umount done on the left side?

Reply:
ah no, it was just needed to copy the logs if i understand correctly, this is the CustomMounts you wanted to add, and the CopyLogs is doing this mount/unmount and moving logs

Reply:
yes, the removed part is the Copylogs hook, which was directly used in the code. Now we just call the hook as it was exactly the smae code repeated